c/array-fpi/brs1.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Benchmarks contributed by Divyesh Unadkat[1,2], Supratik Chakraborty[1], Ashutosh Gupta[1]
+ * [1] Indian Institute of Technology Bombay, Mumbai
+ * [2] TCS Innovation labs, Pune
+ *
+ */
+
+extern void __VERIFIER_error() __attribute__ ((__noreturn__));
+extern void __VERIFIER_assume(int);
+void __VERIFIER_assert(int cond) { if(!(cond)) { ERROR: __VERIFIER_error(); } }
+extern int __VERIFIER_nondet_int(void);
+
+int N;
+
+int main()
+{
+	N = __VERIFIER_nondet_int();
+	int i;
+	int sum[1];
+	int a[N];
+
+	if(N <= 0) return 1;
+
+	for(i=0; i<N; i++)
+	{
+		if(i%1==0)
+		{
+			a[i] = 1;
+		}
+	       	else
```

Comment:
The programs have inconsistent indentation with a mix of tabs and spaces, which makes it hard to read. Of course, proper format is not strictly required, but could make maintenance easier.

Would it be possible to run a code formatter on them and at least ensure absence of spaces and consistent indentation?

Reply:
Let me see what I can do.